the-elder-chores-user-ui/src/app/App.js

```diff
@@ -1,18 +1,45 @@
 import * as React from 'react';
+import { useQuery } from "react-query";
+import { pocketBaseClient } from '../config';
```

Comment:
Could you use the `usePocketClient` hook like in [useCharacterSheet](https://github.com/adventureboss/the-elder-chores/blob/4bdb88b50c40073865f5516317930f0a87060526/the-elder-chores-user-ui/src/services/useCharacterSheet.js#L5)?

Reply:
sure

Reply:
thank you!